spec/models/post_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+
+# Checks if all errors come from given attribute , returns false if otherwise
+def test_attribute_of_post(obj , attribute)
+  # obj : object to check the attributes from
+  # attribute: attribute name of the given object to check errors on
+  obj.valid?
+  error_count = obj.errors.count
+  attribute_err_count = obj.errors[attribute].count
+  if(attribute_err_count < error_count)
```

Comment:
pro tip - > iekš ruby ir boolean tips un tas tev atgriež true vai false, tāpec:

```ruby
return attribute_err_count < error_count
```
ir ekvivalents tavam if.

Un return var nerakstīt, ja tā ir pēdējā instrukcija metodē.

Tavā gadījumā
```ruby
def test_attribute_of_post(obj , attribute)
  obj.valid?
  error_count = obj.errors.count
  attribute_err_count = obj.errors[attribute].count
  attribute_err_count < error_count
end
```

Reply:
`return attribute_err_count < error_count` ar so biju sastapies ari C++ un citas valodas i think tas universali ir , bet par to ka pedeja operacija ari ir return gan ir kinda amazing